bundles/org.openhab.binding.weatherunderground/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -425,6 +477,10 @@
 		<label>Rain Fall Hour</label>
 		<description>Rain fall during the last hour</description>
 		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
```

Comment:
Same here

Reply:
The binding is old, I do not remember if this channel is a real measurement or a forecast.
But if I am not wrong, this binding only provides forecast data.

Reply:
See above

Reply:
`rainHour` is now the only channel with "measurement" rather than "forecast".
I am not sure it is correct. But I am also not sure it is wrong...

Reply:
Indeed. By contrast I cannot imagine how "rain in the last hour" could be called a forecast. So I suggest to leave it as Measurement. And it can be changed later if any user reports it as wrong.